yb-voyager/src/dbzm/dbzm.go

```diff
@@ -102,8 +102,9 @@ func (d *Debezium) Start() error {
 			// In case of logical replication connector we need the path /opt/yb-voyager/debezium-server/yb-connector
 			YB_OR_PG_CONNECTOR_PATH = filepath.Join(DEBEZIUM_DIST_DIR, "yb-connector")
 		} else {
-			// In case of gRPC connector the DEBEZIUM_DIST_DIR is set to debezium-server-1.9.5, hence the connector path is empty to avoid any errors in run.sh
-			YB_OR_PG_CONNECTOR_PATH = ""
+			// In case of gRPC connector the DEBEZIUM_DIST_DIR is set to debezium-server-1.9.5 and the connector is in debezium-server-1.9.5/yb-grpc-connector
+			//This is done to load this jar at the end in the classpath to avoid classpath issues with the jar
```

Comment:
would suggest elaborating a bit here so that we don't forget the context. 
- was failing with sqlite jdbc driver not found error. 
- the connector jar has that service (java.sql.Driver), but the class is not found
- because that class is not found, it seems to error out and does not find the sqlite jdbc driver.